src/pages/profiles/ProfileDetailOverview.tsx

```diff
@@ -1,203 +1,185 @@
-                    >
-                      {name}
-                    </Link>
+    <div className="profile-overview-tab">
+      <Row className="general">
+        <Col size={3}>
+          <h2 className="p-heading--4">General</h2>
+        </Col>
+        <Col size={7}>
+          <table>
+            <tbody>
+              <tr>
+                <th className="p-muted-heading">Name</th>
+                <td>{profile.name}</td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">Description</th>
+                <td>{profile.description ? profile.description : "-"}</td>
+              </tr>
+            </tbody>
+          </table>
+        </Col>
+      </Row>
+      <Row className="devices">
+        <Col size={3}>
+          <h2 className="p-heading--4">Devices</h2>
+        </Col>
+        <Col size={7}>
+          <table>
+            <tbody>
+              <tr className="list-wrapper">
+                <th className="p-muted-heading">Networks</th>
+                <td>
+                  {Object.values(profile.devices).some(isNicDevice) ? (
+                    <ExpandableList
+                      progressive
+                      items={Object.values(profile.devices)
+                        .filter(isNicDevice)
+                        .map((device, i) => (
+                          <div
+                            key={`nic-${i}`}
+                            className="u-truncate list-item"
+                            title={device.network}
+                          >
+                            {device.network}
+                          </div>
+                        ))}
+                    />
+                  ) : (
+                    <>-</>
+                  )}
+                </td>
+              </tr>
+              <tr className="list-wrapper">
+                <th className="p-muted-heading">Storage</th>
+                <td>
+                  {Object.values(profile.devices).some(isDiskDevice) ? (
+                    <ExpandableList
+                      progressive
+                      items={Object.values(profile.devices)
+                        .filter(isDiskDevice)
+                        .map((device, i) => (
+                          <div
+                            key={`disk-${i}`}
+                            className="u-truncate list-item"
+                            title={device.pool}
+                          >
+                            {device.pool}
+                          </div>
+                        ))}
+                    />
+                  ) : (
+                    <>-</>
+                  )}
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </Col>
+      </Row>
+      <Row className="instances list-wrapper">
+        <Col size={3}>
+          <h2 className="p-heading--4">Instances</h2>
+        </Col>
+        <Col size={7}>
+          {usedByInstances.length ? (
```

Comment:
Just another opinion which could be useful. Conditions inside other conditions might be harder to follow. I'd do something like this instead:
```
{usedByInstances.length === 0 && <>-</>}
{usedByInstances.length > 0 && (
    <>
        {isDefaultProject && ...}
        {!isDefaultProject && ...}
    </>
)}
```